config.json

```
@@ -6,6 +6,7 @@
   "magister_url": "",
   "magister_username": "",
   "magister_password": "",
+  "address": "Straat 10, 1234 AA Amsterdam, Nederland",
```

Good idea! There's still a possibility that some classes have a different location. For example, some gym classes could be at an external location. Perhaps you could extend this feature to allow multiple physical locations?

```suggestion
  "address": {
      "default": "Straat 10, 1234 AA Amsterdam, Nederland",
      "some_appointment_title": "Different Address",
      "some_other_title": "Another Address"
```

---

Wouldn't it make more sense to look at the classroom? Par example I have PE in either 005 (default address) or SH1 (which is a different building), however, the title ('lo' in both cases) is still the same.